ffi/opaque/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM google/dart:2.7
```

Comment:
You probably want a newer version of the Dart docker image to get a working FFI.